client/signup/steps/about/index.jsx

```diff
@@ -295,19 +294,37 @@ class AboutStep extends Component {
 		} );
 
 		//Site Goals
-		this.props.setSiteGoals( siteGoalsInput );
-		themeRepo = this.state.hasPrepopulatedVertical
-			? 'pub/radcliffe-2'
-			: getThemeForSiteGoals( siteGoalsInput );
-		designType = getSiteTypeForSiteGoals( siteGoalsInput, this.props.flowName );
-
-		for ( let i = 0; i < siteGoalsArray.length; i++ ) {
-			eventAttributes[ `site_goal_${ siteGoalsArray[ i ] }` ] = true;
-		}
+		if ( shouldHideSiteGoals ) {
```

Comment:
Even if we pass `themeSlugWithRepo` in the segmentation step, we can leave this as a shim to get around the dependency requirement.